internal/karma/karma.conf.js

```diff
@@ -1,5 +1,56 @@
 // Karma configuration
 // GENERATED BY Bazel
+const path = require('path');
+const fs = require('fs');
+const tmp = require('tmp');
+
+// When karma is configured to use Chrome it will look for a CHROME_BIN
+// environment variable. This line points Karma to use puppeteer instead.
+// See
+// https://github.com/karma-runner/karma-chrome-launcher/blob/master/README.md#headless-chromium-with-puppeteer
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+let files = [
+  TMPL_files
+];
+
+const manifestFile = path.join(process.env.TEST_SRCDIR, 'MANIFEST');
+if (fs.existsSync(manifestFile)) {
```

Comment:
if for some misconfiguration this file is missing, will we just silently fail in an odd way? would it be better to rather fail early?

Reply:
The file is expected to be missing in linux and OSX. It should only be present on Windows.